sdk/src/main/java/com/kin/ecosystem/splash/view/SplashActivity.java

```diff
@@ -28,153 +30,155 @@
+		backButton.setOnClickListener(new OnClickListener() {
+			@Override
+			public void onClick(View v) {
+				splashPresenter.backButtonPressed();
+			}
+		});
+	}
+
+	@Override
+	public void attachPresenter(SplashPresenter presenter) {
+		splashPresenter = presenter;
+		splashPresenter.onAttach(this);
+	}
+
+	@Override
+	public void navigateToMarketPlace() {
+		Intent marketplaceIntent = new Intent(this, EcosystemActivity.class);
+		startActivity(marketplaceIntent);
+		finish();
+	}
 
 	@Override
 	public void onBackPressed() {
 		splashPresenter.backButtonPressed();
 	}
 
 	@Override
-    public void navigateBack() {
-        super.onBackPressed();
+	public void navigateBack() {
+		super.onBackPressed();
 		overridePendingTransition(0, R.anim.kinecosystem_slide_out_right);
-    }
+	}
 
-    @Override
-    public void animateLoading() {
-        letsGetStartedBtn.animateLoading();
-        fadeInView(loadingText);
-    }
+	@Override
+	public void animateLoading() {
+		letsGetStartedBtn.animateLoading();
+		fadeInView(loadingText);
+	}
 
-    @Override
-    public void stopLoading(final boolean reset) {
-    	runOnUiThread(new Runnable() {
+	@Override
+	public void stopLoading(final boolean reset) {
+		runOnUiThread(new Runnable() {
 			@Override
 			public void run() {
 				letsGetStartedBtn.stopLoading(reset);
 				fadeOutLoading();
 			}
 		});
-    }
-
-    private void fadeInView(View view) {
-        view.setVisibility(View.VISIBLE);
-        view.startAnimation(fadeIn);
-    }
-
-    private void fadeOutLoading() {
-        loadingText.startAnimation(fadeOutLoading);
-    }
-
-    @Override
-    public void showToast(@Message final int message) {
-        runOnUiThread(new Runnable() {
-            @Override
-            public void run() {
-            	String msg;
-            	switch (message) {
-					case TRY_AGAIN:
-						msg = getString(R.string.kinecosystem_try_again_later);
-						break;
-					default:
-					case SOMETHING_WENT_WRONG:
-						msg = getString(R.string.kinecosystem_something_went_wrong);
-						break;
-				}
-				Toast.makeText(SplashActivity.this, msg, Toast.LENGTH_SHORT).show();
-            }
-        });
-    }
-
-    @Override
-    protected void onDestroy() {
-        super.onDestroy();
-        splashPresenter.onDetach();
-    }
+	}
+
+	private void fadeInView(View view) {
+		view.setVisibility(View.VISIBLE);
+		view.startAnimation(fadeIn);
+	}
+
+	private void fadeOutLoading() {
+		loadingText.startAnimation(fadeOutLoading);
+	}
+
+	@Override
+	public void showToast(@Message final int msg) {
+		runOnUiThread(new Runnable() {
+			@Override
+			public void run() {
+				Toast.makeText(SplashActivity.this, getMessageResId(msg), Toast.LENGTH_SHORT).show();
+			}
+		});
+	}
+
+	@NonNull
+	private @StringRes int getMessageResId(@Message final int msg) {
+		switch (msg) {
+			case TRY_AGAIN:
+				return R.string.kinecosystem_try_again_later;
+			default:
+			case SOMETHING_WENT_WRONG:
```

Comment:
what was changed in this file?

Reply:
only showToast and getMessageResId.
maybe the rest related to indentation.